.github/workflows/ci.yml

```diff
@@ -7,13 +7,17 @@ on:
 
 jobs:
   lint:
-    uses: dvsa/.github/.github/workflows/nodejs-lint.yaml@v3.1.0
+    uses: dvsa/.github/.github/workflows/nodejs-lint.yaml@v3.5.0
+    with:
+      node-version: 20.x
```

Comment:
Minor and feel free to resolve but we can omit this part if we update the `.nvmrc` as `setup-node` and will use that instead.

Reply:
Ah, that's not ideal 😞  Let's keep it explicit then.